core/src/main/java/com/sequenceiq/cloudbreak/service/metrics/CloudbreakMetricService.java

```diff
@@ -77,6 +81,7 @@ public void recordImageCopyTime(Stack stack, long startMillis) {
                 MetricTag.REGION.name(), Optional.ofNullable(stack.getRegion()).orElse("NA")};
 
         long millispassed = System.currentTimeMillis() - startMillis;
+        LOGGER.debug("Image copy duration report: {} ms", System.currentTimeMillis() - startMillis);
```

Comment:
The above line is exactly the same calculation.

Reply:
Thank you (move error ...).